src/utils/storage.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Retrieve specific data stored against the given key from chrome storage
+ *
+ * @returns { Promise<T> | undefined } data stored in Chrome Storage
+ */
+export const get = async <T>(key: string): Promise<T> => {
+  const dataResult = await chrome.storage.local.get([key]);
```

Comment:
in this file where we use `chrome.` should we change it to `browser.` so it supports both chrome and firefox?

Reply:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/storage/StorageArea/get

Reply:
nm. looks like we have to set the browser based on what browser we are using

Reply:
For now Firefox does not complain when storage is accessed via chrome keyword